app/ui/browser/reducers/main-reducers.js

```diff
@@ -41,7 +41,7 @@ export default function basic(state = initialState, action) {
       return createTab(state, action.location);
 
     case types.IPC_COMMAND_OPEN_BOOKMARK:
-      return createTab(state, action.bookmark.url);
+      return createTab(state, action.bookmark.location);
```

Comment:
@linclark headsup that you might get some conflicts against this PR. I think this is pretty close to landing, perhaps even tonight.